lib/utils/job-utils/racadm-tool.js

```diff
@@ -367,5 +347,103 @@ function racadmFactory(
             });
     };
 
+    /**
+     * Returns a promise with the results or errors of enabling IPMI over lan
+     *
+     * @param {string} host
+     * @param {string} user
+     * @param {string} password
+     * @return {promise}
+     */
+    RacadmTool.prototype.enableIpmi = function(host, user, password) {
+        return this.runCommand(host, user, password, "set iDRAC.IPMILan.Enable 1");
+    };
+
+    /**
+     * Returns a promise with the results or errors of disabling IPMI over lan
+     *
+     * @param {string} host
+     * @param {string} user
+     * @param {string} password
+     * @return {promise}
+     */
+    RacadmTool.prototype.disableIpmi = function(host, user, password) {
+        return this.runCommand(host, user, password, "set iDRAC.IPMILan.Enable 0");
+    };
+
+    /**
+     * Returns a promise with the results or errors of configuring one BIOS attribute
+     *
+     * @param {string} host
+     * @param {string} user
+     * @param {string} password
+     * @return {promise}
```

Comment:
Forgot some other arguments.
And it's better to describe the resolved value of promise.